plugins/woocommerce/client/legacy/css/woocommerce.scss

```diff
@@ -2444,17 +2380,28 @@ body:not(.search-results) .twentysixteen .entry-summary {
 
 #coming-soon-footer-banner {
```

Comment:
I only changed the styles starting from this line. The others are automatically formatted by the editor (prettier I think).